nuxt.config.js

```diff
@@ -1,138 +1,171 @@
 /* eslint-disable prettier/prettier */
 
 if (process.env.MODE === 'development' || process.env.NODE_ENV === 'development') {
-  require('dotenv').config()
+    require('dotenv').config()
 }
 
 export default {
-  router: {
-    middleware: ['password-protect']
-  },
-  server: {
-    host: process.env.MODE === 'production' ? '0.0.0.0' : 'localhost', // default: localhost
-    port: 3000 // default: 3000
-  },
-  env: {
-    assetsBaseUrl: process.env.MODE === 'production' || process.env.NODE_ENV === 'production' ? '' : 'http://localhost:1337'
-  },
-  mode: 'universal',
-  /*
-   ** Headers of the page
-   */
-  head: {
-    title: process.env.npm_package_name || '',
-    meta: [
-      { charset: 'utf-8' },
-      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
-      {
-        hid: 'description',
-        name: 'description',
-        content: process.env.npm_package_description || ''
-      }
+    /*
+     ** Router configuration
+     */
+    router: {
+        middleware: ['password-protect'],
+        scrollBehavior: async(to, from, savedPosition) => {
```

Comment:
idéalement, ce serait bien d'implémenter la nouvelle façon de faire de nuxt
https://nuxtjs.org/api/configuration-router#scrollbehavior
risque de dépréciation

`WARN  router.scrollBehavior property is deprecated in favor of using ~/app/router.scrollBehavior.js file`
learn more: https://nuxtjs.org/api/configuration-router#scrollbehavior